test/docker-compose.yml

```diff
@@ -10,5 +10,12 @@ services:
     command: ["/bin/sh", "-c", "freshclam --no-warnings && echo 'TCPSocket 3310' | clamd -c /dev/stdin --foreground"]
     tmpfs:                                                                                                                                                                                                     
     - /var/lib/clamav:uid=1000
+    volumes:
+    - type: bind
```

Comment:
IMO - it's typically bad practice to bind mount a file in docker as default behaviour is to create as a directory if the file doesn't exist leading to 'fun' debugging. Anyway to make this a directory mount? If not it's find as is as it looks like it's only used in testing so YMMV in this case.